doc/governance/accept_recovery.rst

```diff
@@ -17,7 +17,10 @@ A member proposes to recover the network and other members can vote on the propo
         "actions": [
             {
                 "name": "transition_service_to_open",
-                "args": null
+                "args": {
```

Comment:
I think it's better to not include a full, valid, certificate as it's likely to be pasted as is by users for their own service. Perhaps `"-----BEGIN CERTIFICATE-----\n ...  \n-----END CERTIFICATE-----\n"` is sufficient here?

Reply:
I added the complete cert because that's what's done in other places, e.g. `proposals.rst`, not sure if we want this here or not.

Reply:
It's fine to do this, we can't remove every sample value in the doc for fear it'll get copy-pasted. Having realistic examples makes the doc much easier to read.

Besides, there's no way that copy pasted proposal will succeed, so it won't do any harm if they copy-paste this particular sample verbatim.